jujugui/static/gui/src/app/components/deployment-summary/deployment-summary.js

```diff
@@ -18,12 +18,90 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-YUI.add('deployment-summary', function() {
+YUI.add('deployment-summary-classic', function() {
 
-  juju.components.DeploymentSummary = React.createClass({
+  juju.components.DeploymentSummaryClassic = React.createClass({
 
-    propTypes: {
-      summaryClearAction: React.PropTypes.func.isRequired
```

Comment:
I'm assuming that these were left out because we'll be removing this component in the long term anyways so it wasn't worth the effort?